intake/catalog/base.py

```diff
@@ -608,7 +609,7 @@ def fetch_page(self, page_offset):
             raise RemoteCatalogError(
                 "Failed to fetch page of entries {}-{}."
                 "".format(page_offset, page_offset + self._page_size)) from err
-        info = msgpack.unpackb(response.content, **unpack_kwargs)
+        info = msgpack.unpackb(response.content, object_hook=decode_datetime, **unpack_kwargs)
```

Comment:
Could this be added to `unpack_kwargs` instead of added to each call individually?